sematic/db/migrations/20230217235835_add_id_to_users.sql

```diff
@@ -0,0 +1,54 @@
+-- migrate:up
```

Comment:
@tscurtu @augray please pay special attention to this migration.

It adds an `id` column to `users`. Until now `users` were indexed by email. In order to display ownership info i various parts of the app, we are going to have to return user objects. We should obfuscate certain columns (`api_key` is already obfuscated). I think we should hide `email` as well. Users that can see each others run are part of the same org, so they probably know each others address, but I think it's still better practice to not include email addresses in returned payloads.

Changing a foreign key in SQLite requires creating a new table. This migration does that and generates uuid for existing users. The uuid generation method is not perfect but should suffice for the case at hand